infrastructure/aws/cdk/config.py

```diff
@@ -56,7 +57,8 @@ class eoSTACSettings(BaseSettings):
 
     timeout: int = 10
     memory: int = 256
-
+    stac_browser_github_tag: None | str = "v3.1.0"  # if not none, will try to deploy this version of radiant earth stac browser
+    stac_api_custom_domain_name: None | str = "https://stac.eoapi.dev"
```

Comment:
🤔 can we avoid setting a default to our stack here?

Reply:
I could remove the default so that we're always force to explicitly write it in the dotenv file, yes. Is that what you mean? 

However if you're question is: "can't the cdk use the dynamic api gateway url instead, when the stack gets synthesised ?" The answer is unfortunately no, because that url is resolved only at deployment (whereas we need it before, during synth)

Reply:
but if `None` later in the application can we default to either the STAC API gateway url or a custom endpoint for STAC that user would have put?

Reply:
`stac_browser_github_tag` can have a default 👍